packages/server/lib/server.js

```diff
@@ -197,10 +197,8 @@ class Server {
 
       logger.setSettings(config)
 
-      // generate our request instance
-      // and set the responseTimeout
-      // TODO: might not be needed anymore
-      this._request = Request({ timeout: config.responseTimeout })
+      // TODO: does not need to be an instance anymore
```

Comment:
`Request` can now be used statically, no remaining invocations provide options in the constructor

in a future PR, `request.js` and usages can be refactored, which will simplify how we do HTTP requests in general